server/service/src/sync/translations/document.rs

```diff
@@ -149,14 +149,14 @@ impl SyncTranslation for DocumentTranslation {
                 DocumentStatus::Active => LegacyDocumentStatus::Active,
                 DocumentStatus::Deleted => LegacyDocumentStatus::Deleted,
             },
-            owner_name_id,
+            owner_name_id: document.owner_name_id,
```

Comment:
This is the `owner_name_link_id` not the `owner_name_id`... We want to send the actual `name.id` to central

Reply:
It works a bit different here `document.owner_name_id` is actually the `owner_name_id`. See other comment...

Reply:
Ah see it! `owner_name_id: owner_name_join.map(|(_, name_row)| name_row.id)` 😆